### PR TITLE
Catalog my titles

### PR DESCRIPTION
**Pull request by iPR0D1G1**
catalog my titles.
fixed:  #6195

### CODE REVIEW

**Review by iPR0D1G1 — COMMENTED** [review]
reviewed

File: apps/catalog/catalog/src/app/dashboard/title/list/list.component.ts [hunk 1]
```diff
@@ -18,7 +18,8 @@ const columns = {
   'release.year': 'Release Year',
   directors: 'Director(s)',
   views: { value: '# Views', disableSort: true },
```
Comment (pldespaigne) [inline apps/catalog/catalog/src/app/dashboard/title/list/list.component.ts:20]: use quotes ' ' around directors & views.

File: apps/catalog/catalog/src/app/dashboard/title/list/list.component.html [hunk 2]
```diff
@@ -25,9 +25,17 @@ <h1>My Titles</h1>
       <bf-table-filter [source]="movies | filterBy: filterByMovie: { value: filter.value, exclude: 'archived' }" [columns]="columns"
         [initialColumns]="initialColumns" clickable showFilter showPaginator (rowClick)="goToTitle($event)">
         <ng-template colRef="title.international" let-title><b>{{ title }}</b></ng-template>
+        <ng-template colRef="views" let-movie="item"><b>{{movie.id | getViews | async}}</b></ng-template>
+        <ng-template colRef="id" let-titleId>
+          <ng-container *ngIf="titleId | getIncomesFromTitle | async as incomes">
+            {{ incomes.length }}
+            <ng-container *ngIf="incomes.length > 0">
+              ({{ incomes | sum : 'price' | currency:incomes[0].currency }})
+            </ng-container>
```
Comment (pldespaigne) [inline apps/catalog/catalog/src/app/dashboard/title/list/list.component.html:34]: Add an `; else` -> "($0)"

**Review by pldespaigne — CHANGES_REQUESTED** [review]

**Review by iPR0D1G1 — COMMENTED** [review]
reviewed

**Review by pldespaigne — APPROVED** [review]

**Pull request merged**